docs/articles/sdk/configure-qtcreator.rst

```diff
@@ -0,0 +1,52 @@
+Configuring QtCreator to use the SDK
+====================================
+
+After the installation of the SDK and sourcing environment, there are couple of
```

Comment:
What is a "sourcing environment"? Is it to distinguish between the "target" env and the "host" env? If so, should the more traditional term "host" be used?

Reply:
it is like 
``source /opt/pelux_sdk/environment-setup-core2-64-pelux-linux`` on target platform.
this step is already described in ``using-the-sdk-to-crosscompile.rst`` section.

In that article there is a session named ``Sourcing Environment`` so this was reference to that one. Surely i can change it if it is unappropriate/

Reply:
I guess it should have been called: "Sourcing the environment"

Reply:
Change it to "... and sourcing the environment,"

Reply:
and it should be "there are a copple of"

Reply:
You may even say `source`ing the environment, so make it clear you are referring to a command.

Reply:
sure i can change it here/ Do you think it should be also changed in using-the-sdk-to-crosscompile.rst ? 

then i can change it there and add it to this PR as well.

Reply:
For maintenance reasons I'm wondering if we should have a pattern in how-tos where we state "Prerequisites" at the top and just have a list of links to other pages. This would reduce the need to rewrite all these introductory paragraphs we have everywhere which are there to basically just hint at where you are in "some part of a path" kind of.

Like:

Prerequisites
* Install SDK <----- this would just be a link
* Sourcing the environment <----- this would just be a link

Should also mention in this context that the overall design idea here is to have "Instruction" type articles that construct a path to follow in order to achieve something which is larger than a single how-to but smaller than an abstract process. This Instruction would explain what the purpose is and then present a list of how-tos that would make up what needs to be done.

For example it could be that "Working with the image on a target" (or whatever) would be an Instruction that explains the overall purpose, what the reader will accomplish, and then proceed to point out distinct how-tos for e.g. configuring the host, downloading the platform, building an image, building and SDK, sourcing the env, flashing, etc.

The purpose is to avoid ripple effects "upwards" in the SWF texts and between articles of the same "type" as much as possible.

Reply:
@kursatkobya In general, I think we should handle issues found in "old" commits as separate fixes. If you want to fix the issue you mention in using-the-sdk-to-crosscompile.rst, I think it should be a separate PR (and absolutely a separate commit!), since it will be easier to find the origins of the fix afterwards if we do it like this.

Reply:
Also, if there is a section about setting up the environment already, maybe you can add a footnote linking to it, so that the user is made aware of this section in case this is the first time the user encounters `source`.

Reply:
@jonte What do you think about my suggestion above? It sort of collides with what you just wrote about footnotes I believe? My proposal would mean that there is no reason to write a nice introduction but rather align the reader in the context by providing a list of prerequisites.

Thinking about this, the list of prerequisites is also a maintenance issue and should perhaps be simplified to be a link to the Instruction instead?

Reply:
@breakreturn 
There are pros and cons of both approaches. I am not sure which one I prefer, but let me list my views of both:

With footnotes/references in-lined (in general, all through the text), we can make a text which is possible to read "front to back" in a nice manner. This means that someone can take the entire SWF and just read it. This is similar to how a book is usually structured (e.g. you can pick up a chapter in the middle of a book, and expect to be referred back to important concepts that you may have missed if you do so).

With a prerequisites section, we end up with something that is closer to a "knowledge base", in which there is a very clear relation between articles, but it may be a bit drier to read than a book, and may not be as "comfortable" to read "front to back".

Regarding introductory pieces for information already conveyed elsewhere; I agree that we should keep this to a minimum. I would have given a short footnote ("for more info on concept X, see Y") - but that's just my personal opinion.

In all honesty though, I didn't see your original comment, and what I commented to @kursatkobya was just how I would rather see the reference being made :)

I am fine with either way, but we should be consistent. @breakreturn you have the best overview of the SWF - please choose what you prefer and I think both ways will be fine.

Reply:
We can refer back to this thread for a reasoning of our choice later on.

Reply:
I believe i can give the link with a different commit after correcting all grammar typos in this one. If this is okay i will keep this as is the i will add another commit with links and corrections on ``using-the-sdk-to-crosscompile.rst``

Reply:
@jonte My proposal was partially intended to remove the need of having in-lined references since there would be no text. It's very un-book-like in structure as you say, but so far everything points to maintenance being the biggest issue with the SWF. I'd rather have a dry text and slightly un-natural structure than wrong content. The maintenance complexity is where my bias comes from in these things.

Maybe we can re-write the intro text here to be very minimal and only reference the other articles, and leave the final design decision to after we've had more design discussions?